studio-cartogram/fondfolio
# Checkout V2 - Payment

To test:
- Append `&checkoutBeta=1` to the order page. You should see a different checkout button
- Click checkout and follow the steps to checkout
